src/process/namespaces.rs

```diff
@@ -39,8 +45,6 @@ impl Process {
 /// See also the [Process::namespaces()] method
 #[derive(Debug, Clone)]
 pub struct Namespace {
-    /// Namespace type
-    pub ns_type: OsString,
```

Comment:
Also, for better compatibility, I'm inclined to keep the `ns_type` field.  For people who are currently using the `Vec<_>` interface, including the `ns_type` in the struct is more useful, at the expense of a small amount of memory duplication

Reply:
No problem will do